frontend/src/components/board/Toolbar.tsx

```diff
@@ -0,0 +1,228 @@
+        renderInput("fill", "Color", "color")}
+      {selectedObjectStyles?.fontSize !== undefined &&
+        renderInput("fontSize", "Font Size", "number")}
+      <div className="flex items-center space-x-2">
+        <StyledLabel>Font styles</StyledLabel>
+        <Toggle
+          aria-label="Toggle bold"
+          pressed={selectedObjectStyles?.fontWeight === "bold"}
+          onPressedChange={onBoldClick}
+        >
+          <Bold className="h-4 w-4" />
+        </Toggle>
+        <Toggle
+          aria-label="Toggle italic"
+          pressed={selectedObjectStyles?.fontStyle === "italic"}
+          onPressedChange={onItalicClick}
+        >
+          <Italic className="h-4 w-4" />
+        </Toggle>
+        <Toggle
+          aria-label="Toggle underline"
+          pressed={selectedObjectStyles?.underline === true}
+          onPressedChange={onUnderlineClick}
+        >
+          <Underline className="h-4 w-4" />
+        </Toggle>
+      </div>
+    </>
+  );
+
+  const renderRectangleControls = () => (
+    <>
+      {selectedObjectStyles?.fill !== undefined &&
+        renderInput("fill", "Color", "color")}
+
+      {selectedObjectStyles?.stroke !== undefined &&
+        renderInput("stroke", "Stroke Color", "color")}
+
+      {selectedObjectStyles?.strokeWidth !== undefined &&
+        renderInput("strokeWidth", "Stroke Width", "number")}
+      {selectedObjectStyles?.width !== undefined &&
+        renderInput("width", "Width", "number")}
+      {selectedObjectStyles?.height !== undefined &&
+        renderInput("height", "Height", "number")}
+
+      {selectedObjectStyles?.angle !== undefined &&
+        renderInput("angle", "Angle", "number")}
+    </>
+  );
+
+  const renderCircleControls = () => (
+    <>
+      {selectedObjectStyles?.fill !== undefined &&
+        renderInput("fill", "Color", "color")}
+
+      {selectedObjectStyles?.stroke !== undefined &&
+        renderInput("stroke", "Stroke Color", "color")}
+
+      {selectedObjectStyles?.strokeWidth !== undefined &&
+        renderInput("strokeWidth", "Stroke Width", "number")}
+      {selectedObjectStyles?.radius !== undefined &&
+        renderInput("radius", "Radius", "number")}
+      {selectedObjectStyles?.angle !== undefined &&
+        renderInput("angle", "Angle", "number")}
+    </>
+  );
+
+  const renderPathControls = () => (
+    <>
+      {selectedObjectStyles?.stroke !== undefined &&
+        renderInput("stroke", "Stroke Color", "color")}
+      {selectedObjectStyles?.strokeWidth !== undefined &&
+        renderInput("strokeWidth", "Stroke Width", "number")}
+    </>
+  );
+
+  const renderLineControls = () => (
+    <>
+      {selectedObjectStyles?.stroke !== undefined &&
+        renderInput("stroke", "Stroke Color", "color")}
+        {selectedObjectStyles?.width !== undefined &&
+        renderInput("width", "Width", "number")}
+      {selectedObjectStyles?.height !== undefined &&
+        renderInput("height", "Height", "number")}
```

Comment:
Lines should also have angles I guess 🤷‍♂️ 
```suggestion
        renderInput("height", "Height", "number")}
      {selectedObjectStyles?.angle !== undefined &&
        renderInput("angle", "Angle", "number")}
```

Reply:
yea, right